sympy/printing/pretty/tests/test_pretty.py

```diff
@@ -5942,7 +5942,11 @@ def test_PrettyPoly():
 
 def test_issue_6285():
     assert pretty(Pow(2, -5, evaluate=False)) == '1 \n--\n 5\n2 '
-    assert pretty(Pow(x, (1/pi))) == 'pi___\n\\/ x '
+    assert pretty(Pow(x, (1/pi))) == \
+    ' 1 \n'\
+    ' --\n'\
+    ' pi\n'\
+    'x  '
```

Comment:
A question for another reviewer - is this better or worse as:
```suggestion
    assert pretty(Pow(x, (1/pi))) == \
    'π ___\n\'
    '╲╱ x '
```
Are any comments in #6825 relevant?

Reply:
Maybe https://github.com/sympy/sympy/pull/17628#issuecomment-542438582 is also relevant

Reply:
Oh nice find. @oscarbenjamin, assuming your opinion is unchanged feel free to merge.

At some point it might make sense to check that the logic in `pretty` matches `latex`:

https://github.com/sympy/sympy/blob/92f31c3ff460bddc936b3b39c6d4d9d5ec0c8ece/sympy/printing/latex.py#L614-L630

But that's not necessary in this PR.